src/dom/Value.ts

```diff
@@ -118,6 +126,12 @@ export interface Value {
    */
   fields(): [string, Value][];
 
+  /**
+   * For the Struct type, returns an array containing the field name/values pairs in the Struct;
```

Comment:
Like `get()`/`getAll()`, I would describe this function in its relation to `fields()`.